sources/PTYTextView.m

```diff
@@ -142,6 +142,7 @@ @implementation PTYTextView {
     double _charHeightWithoutSpacing;
 
     // NSTextInputClient support
+    NSEvent* _keydownEvent;
     BOOL _inputMethodIsInserting;
```

Comment:
This instance variable is used at  https://github.com/gnachman/iTerm2/pull/279#discussion_r61659026.